test/benchmark/tests/utils/apache_bench.py

```diff
@@ -111,8 +108,6 @@ def extract_metrics(self, connection=None):
         # Clean up right away
         run(f"aws s3 rm --recursive {S3_BUCKET_BENCHMARK_ARTIFACTS}/{temp_uuid}/")
```

Comment:
So it seems like now apache bench only works with s3? I'd rather make this configurable since for whatever reason people may not have access to an AWS account or use a different cloud provider. This script is too important.

Reply:
@msaroufim The benchmarking *automation* assumes that anyone running this as access to an aws account, since the automation uses aws ec2 instances for benchmarking. Anyone with aws account has access to s3. In the current state, we can let this script have aws components, and keep using the handy script to benchmark locally on any machine: https://github.com/pytorch/serve/blob/master/benchmarks/benchmark-ab.py

However, we need to refactor the benchmarking scripts at a single location and update the README, this is being tracked here: https://github.com/pytorch/serve/issues/1207